tests/runtime/variable

```diff
@@ -63,7 +63,25 @@ PROG BEGIN { @["hello", 0] = 0; } i:ms:1 { @["hi", 1] = 1; exit(); }
 EXPECT @[hi, 1]: 1
 TIMEOUT 2
 
-NAME tuple string resize
-PROG BEGIN { @ = ("hello", 0); } i:ms:1 { @ = ("hi", 1); exit(); }
-EXPECT @: (hi, 1)
+NAME map tuple string resize
+PROG BEGIN { @[1] = ("hi", 1); @[1] = ("hellolongstr", 2); @[1] = ("by", 3); exit(); }
```

Comment:
Nit: The third assignment goes back to the size of the first - couldn't that hide potential issues with the second assignment?

Reply:
I think I can get rid of the first assignment as I want to show that a shorter string doesn't have issues when the map type holds a longer string.